.github/workflows/ci.yml

```diff
@@ -13,8 +13,14 @@ jobs:
       uses: actions/setup-ruby@v1
       with:
         ruby-version: 2.6.x
-    - name: Ruby style checks
+    - name: Install required packages
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install libmysqlclient-dev libpq-dev libsqlite3-dev libsasl2-dev
```

Comment:
I don't think we test with sqlite3 in CI right now, although I don't see why it wouldn't work.

What is libsasl2-dev a dependency of?

Reply:
Yep, I can remove sqlite. libsasl2 is for memcached.